### DISCUSSION TITLE
Install system-bridge as Windows service

### DISCUSSION THREAD

**Discussion by DarthSonic** [timmo001/system-bridge#3335] [Ideas]
Optionally install system-bridge as Windows service to be able to send text or commands before login (Home Assistant for example could enter login credentials remotely). Also automations based on windows sensors could run without login.

**Comment by msebald** [comment]
+1

I also looked how to run this as a service. Would be great if this is possible. Having the software running without the user logged in has many advantages. Also it cannot be disabled so easily (maybe it would be good to run it without the icon next to the clock).
